vendor/bat-native-ads/src/bat/ads/internal/account/ad_rewards/ad_rewards_test.cc

```diff
@@ -25,7 +25,7 @@ class BatAdsAdRewardsIntegrationTest : public UnitTestBase {
 };
 
 TEST_F(BatAdsAdRewardsIntegrationTest,
-       GetAdRewardsNotUpdatedDueToLossOfPrecisionIssue15801) {
+       FailedToUpdateAdRewardsDueToLossOfPrecisionIssue15801) {
```

Comment:
Renamed for consistency with `FailedToUpdateAdRewardsDueToNotZeroingUnreconciledEstimatedPendingRewardsIssue16678`